index.php

```diff
@@ -30,23 +30,21 @@
 <section class="container lg:w-1/2 mx-auto flex flex-col gap-5">
 
     <?php
-    if (count($blogs) > 0) {
     foreach ($blogs as $blogpost): ?>
         <article class="p-8 border border-solid rounded-md">
             <div class="flex justify-between items-center flex-col md:flex-row mb-4">
                 <h2 class="text-4xl"><?php echo $blogpost->title; ?></h2>
             </div>
-            <p class="text-2xl mb-2"><?php echo $blogpost->postTime . ' - By ' . $blogpost->authorId; ?></p>
+            <p class="text-2xl mb-2"><?php echo $blogpost->postTime . ' - By ' . $blogpost->username?></p>
             <p class="text-2xl mb-2"><?php echo $blogpost->extract ?></p>
             <div class="flex justify-center">
-
-            <a class="px-3 py-2 mt-4 text-lg bg-indigo-400 hover:bg-indigo-700 hover:text-white transition inline-block rounded-sm" href="singlePost.php?id=<?php echo $blogpost->id?>">View post</a>
-
+                <a class="px-3 py-2 mt-4 text-lg bg-indigo-400 hover:bg-indigo-700 hover:text-white transition inline-block rounded-sm" href="singlePost.php?id=<?php echo $blogpost->id?>">View post</a>
             </div>
         </article>
-    <?php endforeach; }
-    else {echo 'Sorry, no posts found.';}
+    <?php endforeach;
+    if (empty($blogs))
+    {echo 'Sorry, no posts found.';}
         ?>
```

Comment:
Just to be consistent with the foreach, can you use the if/endif syntax here as well please? https://www.w3schools.com/php/keyword_endif.asp

<?php if (empty($blogs)): ?>
Sorry ......
<?php endif; ?>

Reply:
done thanks